pillar.example

```diff
@@ -9,7 +9,11 @@ netweaver:
   sapmnt_inst_media: your_nfs_share_SID_folder
   swpm_folder: your_swpm_folder_relative_to_swpm_media
   sapexe_folder: your_sapexe_sar_folder_relative_to_swpm_media
-
+  create_swap: True
```

Comment:
What do you think if we just skip the `create_swap: True` and put directly and replace the `swap_config` by that parameter.
Just to remove one dependant parameter. The final result would be:
```
create_swap:
      directory: /var/lib/swap
      size: 128
```
I think this would be more robust.

Reply:
Please, if you add these changes document them in the pillar.example file. Say how the parameters have to be used